sensors/avl/sensorhub-driver-avl-911/build.gradle

```diff
@@ -17,6 +17,11 @@ osgi {
   manifest {
     attributes('Bundle-Vendor': 'Botts Innovative Research, Inc.')
     attributes('Bundle-Activator': 'org.sensorhub.impl.sensor.avl.Activator')
+    attributes('Export-Package': 'org.sensorhub.impl.sensor.avl')
+    attributes('Import-Package': '''
```

Comment:
I don't think this is needed since we exclude these packages by default in common.gradle

Reply:
this is needed since we exclude these packages by default in common.gradle. Ok for building and installing.  cdc2b03d777c70987963c2bf817b2c32e7410ddf